README.md

```diff
@@ -2,6 +2,8 @@
 
 # Welcome to documentation repository for Confidential Containers 
 
+[![asciicast](https://asciinema.org/a/eGHhZdQY3uYnDalFAfuB7VYqF.svg)](https://asciinema.org/a/eGHhZdQY3uYnDalFAfuB7VYqF)
```

Comment:
Not sure if it is my setup, but this appears quite large and results in the following text not being on my screen?
I wonder if the embedded demo would be better  little lower down , just above the Further Detail heading?
That way the reader has a chance to get context then see the demo as they scroll down?

Reply:
> Not sure if it is my setup, but this appears quite large and results in the following text not being on my screen?

The rich diff view might look weird, does [this](https://github.com/Jakob-Naucke/documentation/tree/asciinema-ssh) look better?

> I wonder if the embedded demo would be better little lower down , just above the Further Detail heading? That way the reader has a chance to get context then see the demo as they scroll down?

Maybe better, let me move it.

Reply:
I think having moved it to Further Detail section works for me.